scripts/sql_to_wasm_shell_link.py

```diff
@@ -0,0 +1,74 @@
+# Note, this may not handle all special characters
+shell_link_stub = "https://shell.duckdb.org/#queries=v0,"
+# sql = """
+#     install tpch;
+#     load tpch;
+#     call dbgen(sf=0.1);
+#     pragma tpch(7);
+# """
+
+sql = """
+CREATE OR REPLACE TABLE business_metrics (
+    product_line VARCHAR, product VARCHAR, year INTEGER, quarter VARCHAR, revenue integer, cost integer
+);
+INSERT INTO business_metrics VALUES
+    ('Waterfowl watercraft', 'Duck boats', 2022, 'Q1', 100, 100),
+    ('Waterfowl watercraft', 'Duck boats', 2022, 'Q2', 200, 100),
+    ('Waterfowl watercraft', 'Duck boats', 2022, 'Q3', 300, 100),
+    ('Waterfowl watercraft', 'Duck boats', 2022, 'Q4', 400, 100),
+    ('Waterfowl watercraft', 'Duck boats', 2023, 'Q1', 500, 100),
+    ('Waterfowl watercraft', 'Duck boats', 2023, 'Q2', 600, 100),
+    ('Waterfowl watercraft', 'Duck boats', 2023, 'Q3', 700, 100),
+    ('Waterfowl watercraft', 'Duck boats', 2023, 'Q4', 800, 100),
+
+    ('Duck Duds', 'Duck suits', 2022, 'Q1', 10, 10),
+    ('Duck Duds', 'Duck suits', 2022, 'Q2', 20, 10),
+    ('Duck Duds', 'Duck suits', 2022, 'Q3', 30, 10),
+    ('Duck Duds', 'Duck suits', 2022, 'Q4', 40, 10),
+    ('Duck Duds', 'Duck suits', 2023, 'Q1', 50, 10),
+    ('Duck Duds', 'Duck suits', 2023, 'Q2', 60, 10),
+    ('Duck Duds', 'Duck suits', 2023, 'Q3', 70, 10),
+    ('Duck Duds', 'Duck suits', 2023, 'Q4', 80, 10),
+
+    ('Duck Duds', 'Duck neckties', 2022, 'Q1', 1, 1),
+    ('Duck Duds', 'Duck neckties', 2022, 'Q2', 2, 1),
+    ('Duck Duds', 'Duck neckties', 2022, 'Q3', 3, 1),
+    ('Duck Duds', 'Duck neckties', 2022, 'Q4', 4, 1),
+    ('Duck Duds', 'Duck neckties', 2023, 'Q1', 5, 1),
+    ('Duck Duds', 'Duck neckties', 2023, 'Q2', 6, 1),
+    ('Duck Duds', 'Duck neckties', 2023, 'Q3', 7, 1),
+    ('Duck Duds', 'Duck neckties', 2023, 'Q4', 8, 1),
+;
+
+FROM business_metrics;
+
+INSTALL pivot_table from community;
+LOAD 'https://community-extensions.duckdb.org/v1.1.1/wasm_eh/pivot_table.duckdb_extension.wasm';
```

Comment:
```suggestion
LOAD 'pivot_table';
```
I had not seen this script, this is amazing!